Amplify/Categories/DataStore/Query/ModelKey.swift

```diff
@@ -56,6 +56,11 @@ extension CodingKey where Self: ModelKey {
         return key.contains(value)
     }
 
+    // MARK: - not contains
+    public func notContains(_ value: String) -> QueryPredicateOperation {
```

Comment:
This represents a substring check, right? In other words, it's not meant to also be used for an equivalent to SQL `WHERE NOT IN (<some list or set>)`?

Reply:
That's correct. It only support `String` checking by doing a `NOT LIKE %value%` ([not like](https://github.com/aws-amplify/amplify-swift/pull/2789/files/68b61be6487a40669eb943f208f8874850c9a886#diff-c2beb770af448b950a26b1bde597b3840e5bcdb6c7b3b263bdff2ea18717bfc2R33)) ([%value%](https://github.com/aws-amplify/amplify-swift/pull/2789/files/68b61be6487a40669eb943f208f8874850c9a886#diff-c2beb770af448b950a26b1bde597b3840e5bcdb6c7b3b263bdff2ea18717bfc2R53)).